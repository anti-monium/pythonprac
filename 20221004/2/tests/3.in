(1, 4), (17, 3), (15, 55), (1, 1), (16, 55), (0, 0), (14, 87), (33, 22), (11, 111), (97, 13), (6, 3), (77, 88)
