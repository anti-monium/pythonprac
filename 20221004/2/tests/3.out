((11, 111), (97, 13), (77, 88))
